packages/calendar/src/LionCalendar.js

```diff
@@ -413,6 +415,12 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
     }
   }
 
+  __selectedDateChanged() {
+    if (!this.selectedDate && this.__initialCentralDate) {
+      this.centralDate = this.__initialCentralDate;
+    }
```

Comment:
Think this is the right direction. I think there was also some complaint about the selectedDate not syncing when programatically set. We could add that here as well if needed

Reply:
There is a test to prevent this from happening: https://github.com/ing-bank/lion/blob/master/packages/calendar/test/lion-calendar.test.js#L144 
Can you remember why we have this test?

Reply:
It makes sense that it doesn;t update while opened. It should however sync next time it's opened: https://jsfiddle.net/4Lbzyc1m/4/.  (open with chromium)

Reply:
So it's safe to say that the calendar shouldn't do this by default, but the calendar should provide an interface to be used by the datepicker, so that:
```
if (!datepicker.opened) calender.syncCentralDateWithSelectedDate()
```